File: 30-Days-of-Code/Day-2-Operators/Clojure/solution.clj
(def mealCost (read-string (read-line)))
(def tipPercent (read-string (read-line)))
(def taxPercent (read-string (read-line)))

(def tip (* mealCost (/ tipPercent 100.0)))
(def tax (* mealCost (/ taxPercent 100.0)))
(def totalCost (+ mealCost tax tip))

(def roundedTotalCost (Math/round totalCost))

(println "The total meal cost is" roundedTotalCost "dollars.")

File: 30-Days-of-Code/Day-2-Operators/Lua/solution.lua
mealCost = io.read("*line")
tipPercent = tonumber(io.read("*line"))
taxPercent = tonumber(io.read("*line"))

tip = mealCost * (tipPercent / 100.0)
tax = mealCost * (taxPercent / 100.0)
totalCost = mealCost + tip + tax

roundedTotalCost = math.floor(totalCost + 0.5)

print(string.format("The total meal cost is %d dollars.\n", roundedTotalCost))
